Prox/Prox/Widgets/ReviewScoreView.swift

```diff
@@ -41,9 +36,9 @@ class ReviewScoreView: UIStackView {
 
         for index in 0..<5 {
             let scoreItem = ReviewScoreItemView()
-            scoreItem.color = color
             scoreItem.backgroundColor = .clear
-            scoreItem.fill = index < score
+            let scoreLeft = self.score - Float(index)
+            setFillScore(score: scoreLeft, forView: scoreItem)
```

Comment:
Rather than duplicating this code, should we call `updateReviewScore` after the for loop?